libs/internal/src/network/http_requester.cpp

```diff
@@ -1,12 +1,24 @@
 #include <boost/url.hpp>
 
+#include <algorithm>
+#include <cctype>
```

Comment:
Nothing is actually from this header?

Reply:
Woops, remains of trying to fix it with `std::transform`.